addons/web/static/src/views/form/form_controller.js

```diff
@@ -93,6 +96,15 @@ export class FormController extends Component {
         this.router = useService("router");
         this.user = useService("user");
         this.viewService = useService("view");
+        this.ui = useService("ui");
+        const renderOnResize = useDebounced(this.render, 101);
+        useEffect(
+            () => {
+                browser.addEventListener("resize", renderOnResize);
+                return () => browser.removeEventListener("resize", renderOnResize);
+            },
+            () => []
+        );
```

Comment:
should the ui service trigger an event on the bus when the size class changes?

Reply:
As noted in my other comment, I think the UI service should expose a reactive on which people can call useState. The 101 earlier is because we want to be guaranteed that it will be called *after* the ui service has updated the size, and the ui service uses a debounce of 100. I really don't like having to do this.

Reply:
Even though it's not the perfect, long-term, reactive, solution, triggering an event in the ui bus when the size change would be an easy, low cost workaround

Reply:
Agreed and done.